dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -25,691 +16,343 @@
 import com.dotmarketing.portlets.contentlet.model.ContentletVersionInfo;
 import com.dotmarketing.portlets.htmlpageasset.model.IHTMLPage;
 import com.dotmarketing.services.PageServices;
-import com.dotmarketing.util.InodeUtils;
 import com.dotmarketing.util.Logger;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 import com.google.common.collect.Lists;
 
 /**
- * This class provides utility routines to interact with the Multi-Tree
- * structures in the system. A Multi-Tree represents the relationship between a
- * Legacy or Content Page, a container, and a contentlet.
+ * This class provides utility routines to interact with the Multi-Tree structures in the system. A
+ * Multi-Tree represents the relationship between a Legacy or Content Page, a container, and a
+ * contentlet.
  * <p>
- * Therefore, the content of a page can be described as the sum of several
- * Multi-Tree records which represent each piece of information contained in it.
+ * Therefore, the content of a page can be described as the sum of several Multi-Tree records which
+ * represent each piece of information contained in it.
  * </p>
  * 
  * @author will
  */
 public class MultiTreeFactory {
-    
-    private static final String DELETE_MULTITREE_ERROR_MSG = "Deleting MultiTree Object failed:";
-    private static final String SAVE_MULTITREE_ERROR_MSG = "Saving MultiTree Object failed:";
 
-	public static void deleteMultiTree(Object o1, Object o2, Object o3) {
-		Inode inode1 = (Inode) o1;
-		Inode inode2 = (Inode) o2;
-		Inode inode3 = (Inode) o3;
+    final static String DELETE_SQL = "delete from multi_tree where parent1=? and parent2=? and child=? and  relation_type = ?";
+    final static String SELECT_SQL =
+            "select * from multi_tree where parent1 = ? and parent2 = ? and child = ? and  relation_type = ?";
 
-		try {
+    final static String INSERT_SQL =
+            "insert into multi_tree (parent1, parent2, child, relation_type, tree_order ) values (?,?,?,?,?)  ";
 
+    final static String SELECT_BY_ONE_PARENT = "select * from multi_tree where parent1 = ? or parent2 = ? ";
+    final static String SELECT_BY_TWO_PARENTS = "select * from multi_tree where parent1 = ? and parent2 = ?  order by tree_order";
+    final static String SELECT_ALL = "select * from multi_tree  ";
+    final static String SELECT_BY_CHILD = "select * from multi_tree where child = ?  order by parent1, parent2, relation_type ";
+    final static String SELECT_BY_PARENTS_AND_RELATIONS =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)